test/connectors/CustomsDeclareExportsMovementsConnectorSpec.scala

```diff
@@ -69,14 +74,124 @@ class CustomsDeclareExportsMovementsConnectorSpec extends ConnectorSpec {
+           |    "requestTimestamp":"${expectedSubmission.requestTimestamp}"
+           |  }
+           |]""".stripMargin
+
+      stubFor(
+        get(s"$FetchAllSubmissionsEndpoint?providerId=$providerId")
+          .willReturn(aResponse().withStatus(OK).withBody(submissionsJson))
+      )
+
+      val response = connector.fetchAllSubmissions(providerId).futureValue
+
+      val expectedUrl = s"$FetchAllSubmissionsEndpoint?providerId=$providerId"
+      verify(getRequestedFor(urlEqualTo(expectedUrl)))
+
+      response mustBe Seq(expectedSubmission)
+    }
+  }
+
+  "fetch single Submission" should {
+
+    "send GET request to the backend" in {
+
+      val expectedSubmission = exampleSubmissionFrontendModel()
+      val submissionJson =
+        s"""
+           |  {
+           |    "uuid":"${expectedSubmission.uuid}",
+           |    "eori":"$validEori",
+           |    "conversationId":"$conversationId",
+           |    "ucrBlocks":[
+           |      {
+           |        "ucr":"$correctUcr",
+           |        "ucrType":"D"
+           |      }
+           |    ],
+           |    "actionType":"Arrival",
+           |    "requestTimestamp":"${expectedSubmission.requestTimestamp}"
+           |  }
+           |""".stripMargin
+
+      stubFor(
+        get(s"$FetchSingleSubmissionEndpoint/$conversationId?providerId=$providerId")
+          .willReturn(aResponse().withStatus(OK).withBody(submissionJson))
+      )
+
+      val response = connector.fetchSingleSubmission(conversationId, providerId).futureValue
+
+      val expectedUrl = s"$FetchSingleSubmissionEndpoint/$conversationId?providerId=$providerId"
+      verify(getRequestedFor(urlEqualTo(expectedUrl)))
+
+      response mustBe Some(expectedSubmission)
+    }
+  }
+
+  "fetch Notifications" should {
+
+    "send GET request to the backend" in {
+
+      val expectedNotification = exampleNotificationFrontendModel()
+      val notificationsJson =
+        s"""[
+          |   {
+          |     "timestampReceived":"${expectedNotification.timestampReceived}",
+          |     "conversationId":"$conversationId",
+          |     "responseType":"${ControlResponse.value}",
+          |     "entries":[
+          |       {
+          |         "ucrBlock":{
+          |           "ucr":"$correctUcr",
+          |           "ucrType":"D"
+          |         },
+          |         "goodsItem":[]
+          |       }
+          |     ],
+          |     "errorCodes":[],
+          |     "messageCode":""
+          |   }
+          |]""".stripMargin
+
+      stubFor(
+        get(s"$FetchNotifications/$conversationId?providerId=$providerId")
+          .willReturn(aResponse().withStatus(OK).withBody(notificationsJson))
+      )
+
+      val response = connector.fetchNotifications(conversationId, providerId).futureValue
+
+      val expectedUrl = s"$FetchNotifications/$conversationId?providerId=$providerId"
+      verify(getRequestedFor(urlEqualTo(expectedUrl)))
```

Comment:
As mentioned before, think its better just to do 
```
verify(
    getRequestedFor(urlEqualTo("/submissions/321?providerId=123"))
)
```
That way we test the contract